css/master-styles.css

```diff
@@ -104,6 +104,12 @@ h1 {margin: 10px 0px 20px 0px}
 h2 {margin: 10px 0px 15px 0px}
 h3 {margin: 15px 0px 15px 0px}
 
+
+li {
+	margin: 10px 10px 5px 30px;
+
+}
+
 * {
```

Comment:
I would move this up to the top right below where the reset ends. This is a global rule that will affect every item on the page.